astropy/units/si.py

```diff
@@ -436,6 +455,20 @@
     prefixes=True,
     doc="curie: unit of radioactivity",
 )
+def_unit(
+    ["Gy", "gray"],
+    m**2 * s**-2,
```

Comment:
I'd tend to write this as "J * kg**-1", which I think makes the intent clearer.

Reply:
done as suggested